examples/configs/baroclinic_c12.yaml

```diff
@@ -1,6 +1,6 @@
 stencil_config:
   compilation_config:
-    backend: numpy
+    backend: dace:cpu
```

Comment:
Do we need both numpy and dace:cpu here?  Or do we have other tests for numpy that provide adequate coverage?

Reply:
Sorry this was a relic from the testing the orchestration issues. Will keep it at numpy only